src/components/programs/RenameProgram.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+	HStack,
+	FormControl,
+	FormLabel,
+	Input,
+	useDisclosure,
+	Icon,
+	Text,
+	useBreakpointValue,
+	useColorMode,
+	useColorModeValue
+} from '@chakra-ui/react';
+
+import { BsPencil } from 'react-icons/bs';
+
+import type { IPCProgram } from 'types/types';
+
+import { textColorMode } from 'config/colorMode';
+
+
+import Modal from 'components/Modal';
+import Button from 'components/Button';
+
+import { ChangeEvent, useState } from 'react';
+
+/*
+import { useUserContext } from 'contexts/user';
+import { useDriveContext } from 'contexts/drive';
+*/
+
+
+type RenameProgramProps = {
+	program: IPCProgram;
+};
+
+const RenameProgram =  ({ program}: RenameProgramProps): JSX.Element =>  {
```

Comment:
Looks like you still need to install Prettier correctly, it's not formatted here
```suggestion
const RenameProgram = ({ program }: RenameProgramProps): JSX.Element =>  {
```